web/templates/settings/playground_settings_admin.hbs

```diff
@@ -2,24 +2,35 @@
     <div class="admin-table-wrapper">
         <p>
             {{#tr}}
-                Configure external code playgrounds for your Zulip organization. Code playgrounds are interactive in-browser development
-                environments, such as <z-link-repl>replit</z-link-repl>, that are designed to make it convenient to edit and debug
-                code. Zulip code blocks that are <z-link-markdown-help>tagged with a programming language</z-link-markdown-help> will have
-                a button visible on hover that allows you to open the code block in the code playground site.
+                Configure external <z-link-code-playgrounds>code playgrounds</z-link-code-playgrounds>,
+                such as <z-link-repl>replit</z-link-repl>, so that <z-link-code-blocks>code blocks</z-link-code-blocks>
+                that are tagged with a programming language will have a button visible on
+                hovering that allows users to open the code block on the code playground site.
+                {{#*inline "z-link-code-playgrounds"}}<a href="/help/code-blocks#code-playgrounds" target="_blank" rel="noopener noreferrer">{{> @partial-block}}</a>{{/inline}}
                 {{#*inline "z-link-repl"}}<a href="https://replit.com/" target="_blank" rel="noopener noreferrer">{{> @partial-block}}</a>{{/inline}}
-                {{#*inline "z-link-markdown-help"}}<a href="/help/code-blocks" target="_blank" rel="noopener noreferrer">{{> @partial-block}}</a>{{/inline}}
+                {{#*inline "z-link-code-blocks"}}<a href="/help/code-blocks" target="_blank" rel="noopener noreferrer">{{> @partial-block}}</a>{{/inline}}
             {{/tr}}
         </p>
         <p>
             {{#tr}}
-            For example, to configure code playgrounds for languages like Python or JavaScript, you could specify the <i>Language</i>
-            and <i>URL prefix</i> fields as:
+            For example, to configure a code playground for code blocks tagged with
+            JavaScript, you could specify the language and URL prefix fields as:
             {{/tr}}
         </p>
         <ul>
-            <li><code>Python</code> {{t "and" }} <code>https://replit.com/languages/python3/?code=</code></li>
-            <li><code>JavaScript</code> {{t "and" }} <code>https://replit.com/languages/javascript/?code=</code></li>
+            <li>
+                {{t "Language" }}: <span class="rendered_markdown"><code>JavaScript</code></span>
+            </li>
+            <li>
+                {{t "URL prefix" }}: <span class="rendered_markdown"><code>https://replit.com/languages/javascript/?code=</code></span>
+            </li>
         </ul>
+        <p>
+            {{#tr}}
+            The name field appears in the tooltip shown when a user hovers on
+            the button in the code block.
+            {{/tr}}
+        </p>
```

Comment:
I think we should use the same example as in the placeholder text. Otherwise, you can have the annoying experience that you want the playground in the placeholder, but don't have a copy-paste way to set it up.

Reply:
Done.